go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/openshift/microshift
 go 1.17
 
 require (
-	github.com/coreos/go-systemd v0.0.0-20190321100706-95778dfbb74e // route-controller-manager
+	github.com/coreos/go-systemd v0.0.0-20190321100706-95778dfbb74e // microshift
```

Comment:
rcm lost this dependency https://github.com/openshift/route-controller-manager/pull/9, so now it is just a direct dependency of microshift